_maps/map_files/Blueshift/BlueShift.dmm

```diff
@@ -98975,7 +99003,7 @@
 "oJb" = (
 /obj/effect/spawner/structure/window/reinforced,
 /obj/machinery/door/poddoor/preopen{
-	id = "Xeno11";
+	id = ''Xeno11'';
```

Comment:
Make sure you don't have any of these

Reply:
i tried to find that one but i could not find where the object in question was

Reply:
wait i can just use notepad++ probably